### ISSUE TITLE
[Mobile]: OTP verification via Firebase does not work in Sierra Leone

### ISSUE THREAD

**Issue by KarinBerg** [socialincome-san/public#952]
### Describe the bug

If a user in Sierra Leone tries to verify its phone number via the app, then Firebase through a FirebaseAuthException and no OTP is send to the user.

The whole error message is: "FirebaseAuthException: [firebase_auth/unknown] An internal error has occurred. [ Error code:39 ]" in signup_cubit.dart in SignupCubit.signupWithPhoneNumber.<fn> in Zeile 49.

The issue started around the 11th September 2024. 

Examples of failing accounts are:

Phone 1: +23275588647
GMT Time: 12:38, 16. Sept 2024
Test: failed (Error code 39)
Provider: Orange SL

Phone 2: +23278365433
GMT Time: 07:41, 17. Sept 2024
Test: failed (Internal error)
Provider: Orange SL

Phone 3: +23278303080
GMT Time: 07:28, 17. Sept 2024
Test: failed (Error code 39)
Provider: Orange SL

### Desktop Environment (please complete the following information)

```markdown
- OS: Android and iOS
- Browser:
- Version:
```

### Smartphone Type (please complete the following information)

```markdown
- Device:
- OS:
- Browser:
- Version:
```

### Code of Conduct

- [x] I've read the Code of Conduct and understand my responsibilities as a member of the Social Income community

**Comment by KarinBerg** [comment]
I would suggest, that we switch to https://www.twilio.com/ for phone number verification.
Requirement would be that we ensure that twilio is working in Sierra Leone.

**Comment by ssandino** [comment]
Yes, we can confirm that it works: https://www.twilio.com/en-us/sms/pricing/sl
